.github/workflows/docker-build-and-push-arm64.yaml

```diff
@@ -48,6 +48,11 @@ jobs:
           (github.event_name == 'push' && github.ref_type == 'tag') }}
         uses: ./.github/actions/free-disk-space
 
+      - name: Set Swap Space
```

Comment:
We probably would like to add this to  `docker-build-and-push-cuda` job as well.

Reply:
Good catch! https://github.com/autowarefoundation/autoware/pull/5688/commits/dacdc2ea270916eea617139df735157f9b085149